extra-scientific/vtk/autobuild/prepare

```diff
@@ -0,0 +1,3 @@
+abinfo 'Patching required version number to use our patched external libraries ...'
+sed -e "s|2.4.0|2.3.0|" -i "$SRCDIR"/ThirdParty/libharu/CMakeLists.txt
+sed -e "s|8.1.0|8.0.1|" -i "$SRCDIR"/ThirdParty/fmt/CMakeLists.txt
```

Comment:
Seems like a very fragile substitution, I would suggest generating a patch. That way, if the upstream package updates, the next maintainer would notice the issue.